retriever/__main__.py

```diff
@@ -31,7 +32,13 @@
 
 
 def main():
-    """This function launches the Data Retriever."""
+    """This function launches the Data Retriever."""  
+    logging.basicConfig(
+        format="%(asctime)s [%(levelname)-5.5s] [%(filename)s:%(lineno)s - %(funcName)3s() ] %(message)s",
+        handlers=[
+            logging.FileHandler("retriever.log"),
```

Comment:
Let's avoid hardcoding this file `retriever.log` since it's universal, we can include it in the defaults `retriever.lib.defaults`